ResidentInformationApi.Tests/V1/Helper/TestHelper.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web;
+using AutoFixture;
+using Moq;
+using Moq.Protected;
+
+namespace ResidentInformationApi.Tests.V1.Helper
+{
+    public static class TestHelper
+    {
+        public static void SetUpMessageHandlerToReturnJson(Mock<HttpMessageHandler> messageHandler, string rqpString = null, string expectedJsonString = null)
```

Comment:
Nice refactoring!

Reply:
Thanks, I was able to find some similar test helpers to base this off